sb-utils/src/main/java/org.sunbird/ActorServiceException.java

```diff
@@ -1,22 +1,26 @@
 package org.sunbird;
 
+import java.util.Locale;
+import org.sunbird.message.IResponseMessage;
+import org.sunbird.message.Localizer;
+
 public class ActorServiceException {
 
   public static class InvalidOperationName extends BaseException {
-    public InvalidOperationName(String code, String message, int responseCode) {
-      super(code, message, responseCode);
-    }
-  }
-
-  public static class InvalidRequestTimeout extends BaseException {
-    public InvalidRequestTimeout(String code, String message, int responseCode) {
-      super(code, message, responseCode);
+    public InvalidOperationName(Locale locale) {
```

Comment:
we must have two constructors or default locale = null in just one. Passing null isn't nice.